content/compose/compose-file/05-services.md

```diff
@@ -1698,6 +1708,8 @@ userns_mode: "host"
 
 ## uts
 
+{{< introduced compose 2.15.1 "../release-notes.md#2151" >}}
```

Comment:
I also didn't see the note under the subheading for this in the [deploy preview](https://deploy-preview-19773--docsdocker.netlify.app/compose/compose-file/05-services/#uts) 🤔